Sources/ArcGISToolkit/Components/Form/FormInputs/DateTimeInput.swift

```diff
@@ -66,30 +60,30 @@ struct DateTimeInput: View {
             InputFooter(element: element, requiredValueMissing: requiredValueMissing)
         }
         .padding([.bottom], elementPadding)
+        .onChange(of: model.focusedFieldName) { newFocusedFieldName in
+            isEditing = newFocusedFieldName == element.fieldName
+        }
         .onAppear {
-            if inputModel.value.isEmpty {
+            if inputModel.formattedValue.isEmpty {
                 date = nil
             } else {
-                date = try? Date(inputModel.value, strategy: .arcGISDateParseStrategy)
+                date = try? Date(inputModel.formattedValue, strategy: .arcGISDateParseStrategy)
```

Comment:
For `DateTimeInput`s, `inputModel.value` is already a date.

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/0fec9d87-6b3b-4d67-9a2c-cd192ab1c235)

Reply:
Ha ha, looks like you already changed this one! Nice!

Reply:
Yep I had my eye on it and revisited this morning.